core/src/main/java/nl/nn/adapterframework/scheduler/job/SendMessageJob.java

```diff
@@ -55,19 +57,16 @@ public void configure() throws ConfigurationException {
 
 	@Override
 	public void execute() throws JobExecutionException, TimeoutException {
-		try {
-			localSender.open();
-			//sendMessage message cannot be NULL
-			Message message = new Message((getMessage()==null) ? "" : getMessage());
-			PipeLineSession session = new PipeLineSession();
+		try (Message toSendMessage = new Message((getMessage() == null) ? "" : getMessage());
```

Comment:
Misschien dat het een verbetering is als we een NullMessage zouden gebruiken in plaats van een leeg message.
Ik denk voor nu even buiten scope, maar wel een TODO waardig